mycroft/client/speech/hotword_factory.py

```diff
@@ -326,6 +332,10 @@ def initialize():
                     hotword, module
                 ))
                 instance = None
+            except PreciseUnavailable:
+                LOG.warning('Settings prevent Precise Engine use,
+                            falling back to default.')
```

Comment:
Hmm this multiline string doesn't seem to be valid. I suggest using

```python
        LOG.warning('Settings prevent Precise Engine use, '
                    'falling back to default.')
```

Reply:
Huh, thought that was what I changed it to already. Should be fixed now.